docs/kafka/topic-configuration-kafka/README.adoc

```diff
@@ -339,19 +335,41 @@ Maximum message timestamp difference is the maximum difference allowed between t
 [cols="25%,75%"]
 |===
 h|Editable
-|No
+|Yes
 
 h|Type
 |Long
 
 h|Default value
-|`9223372036854775807 ms`
+|`9223372036854775807` (milliseconds)
 
 h|Kafka property name
 |`message.timestamp.difference.max.ms`
 |===
 --
 
+Message down-conversion::
++
+--
+Message down-conversion determines whether the broker can convert the `message.format.version` property value to an older version for consumers that require an older message format version. By default, this property is enabled in order to avoid an `UNSUPPORTED_VERSION` error for consumption requests from older clients. If this property adds excessive load to your broker, you can disable it as needed.
```

Comment:
Can remove "as needed".

Reply:
Yeah, I see what you're saying, but I think _excessive_ already implies that this high load is unwanted.

Reply:
Sure thing. Removed.